### PR TITLE
feature(core): redesign browse auctions

### LINKED ISSUE #1879: `browse auction` - update UI
#### What UI/UX improvements would you like to be implemented: 
- Update UI to match the following 
- ~~To remove the beta feature banner~~
- Include bid status

#### Which component/screens does this concern?:
- Auction screen

#### Anything else we need to know?:
image

The sort order of auctions:
image

### PR DESCRIPTION
**Pull request by pierregee**
#### What kind of PR is this?:

/kind feature

#### What this PR does / why we need it:
- Redesign auction cards
- Add filtering - From your vault
- Add filtering - With placed bids
- Add sorting logic in the ff order:
        - Placed bid lost (if multiple time remaining)
        - Placed bid winning
        - From your vault
        - Time remaining 

#### Which issue(s) does this PR fixes?:

Fixes #1879

#### Developer Checklist:

- [x] Read your code changes at least once
- [x] Tested on iOS/Android device (e.g, No crashes, library supported etc.)
- [x] No console errors on web
- [x] Tested on Light mode and Dark mode*
- [x] Your UI implementation visually matched the rendered design*
- [x] Unit tests*
- [x] Added e2e tests*
- [ ] Added translations*

### CODE REVIEW

**Review by thedoublejay — CHANGES_REQUESTED** [review]
There's a significant performance lag with this update. Maybe we need to re-check. 
1. Switching between Auctions, DEX and other tabs 
2. Switching to ALL tabs is causing slowness

File: mobile-app/app/screens/AppNavigator/screens/Auctions/components/BatchCard.tsx [hunk 1]
```diff
@@ -112,10 +118,42 @@ export function BatchCard (props: BatchCardProps): JSX.Element {
             </View>
           </View>
           <View style={tailwind('flex flex-row items-center justify-center')}>
-            <CollateralTokenIconGroup
-              symbols={batch.collaterals.map(collateral => collateral.displaySymbol)}
-              maxIconToDisplay={3}
-            />
+            {batch?.highestBid?.owner !== address && batch.froms.some(bidder => bidder === address) &&
```
Comment (thedoublejay) [inline mobile-app/app/screens/AppNavigator/screens/Auctions/components/BatchCard.tsx:121]: Should we store this on a variable and maybe memoized it? So it doesn't recheck on every re-render

**Review by thedoublejay — CHANGES_REQUESTED** [review]

File: mobile-app/app/screens/AppNavigator/screens/Auctions/components/BrowseAuctions.tsx [hunk 2]
```diff
@@ -179,28 +192,73 @@ function BatchCards ({
...
+        id: AuctionTabGroupKey.WithPlacedBids,
+        label: translate('screens/BrowseAuctions', 'With placed bids'),
+        handleOnPress: () => onButtonGroupChange(AuctionTabGroupKey.WithPlacedBids),
+        widthPercentage: new BigNumber(42)
+      }
+    ]
+
+    return (
+      <>
+        {isBetaFeature('auction') &&
```
[the remark this answers is not in the document]
Comment (pierregee) [reply mobile-app/app/screens/AppNavigator/screens/Auctions/components/BrowseAuctions.tsx]: Removed in another PR

**Comment by pierregee** [comment]
The design has changed for auctions v2

**Pull request closed without merging**